action.py

```diff
@@ -315,254 +352,275 @@ def _test_package(self, include, data):
+                    r"^(\*|<=?\d{4}|>=?\d{4}|\d{4} - \d{4})$",
+                    "sublime_text must be `*`, of the form "
+                    "`<relation><version>` "
+                    "where <relation> is one of {<, <=, >, >=} "
+                    "and <version> is a 4 digit number, "
+                    "or of the form `<version> - <version>`",
+                )
+
+            elif k == "platforms":
                 if isinstance(v, str):
                     v = [v]
                 for plat in v:
-                    self.assertRegex(plat,
-                                     r"^(\*|(osx|linux|windows)(-(x(32|64)|arm64))?)$")
+                    self.assertRegex(plat, r"^(\*|(osx|linux|windows)(-(x(32|64)|arm64))?)$")
 
-                self.assertCountEqual(v, list(set(v)),
-                                      "Specifying the same platform multiple times is redundant")
+                self.assertCountEqual(
+                    v,
+                    list(set(v)),
+                    "Specifying the same platform multiple times is redundant",
+                )
 
-                if (("osx-x32" in v and "osx-x64" in v and "osx-arm64" in v) or
-                    ("windows-x32" in v and "windows-x64" in v and "windows-arm64" in v) or
-                    ("linux-x32" in v and "linux-x64" in v and "linux-arm64" in v)):
+                if (
+                    ("osx-x32" in v and "osx-x64" in v and "osx-arm64" in v)
+                    or ("windows-x32" in v and "windows-x64" in v and "windows-arm64" in v)
+                    or ("linux-x32" in v and "linux-x64" in v and "linux-arm64" in v)
+                ):
                     self.fail("Specifying all of x32, x64 and arm64 architectures is redundant")
 
-                self.assertFalse(set(["osx", "windows", "linux"]) == set(v),
-                                 '"osx, windows, linux" are similar to (and should be replaced by) "*"')
+                self.assertFalse(
+                    {"osx", "windows", "linux"} == set(v),
+                    '"osx, windows, linux" are similar to (and should be replaced by) "*"',
+                )
 
-            elif k == 'date':
+            elif k == "date":
                 self.assertRegex(v, r"^\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d$")
 
-            elif k == 'tags':
-                self.assertTrue(bool(v),
-                                '"tags" must be `true` or a string of length>0')
+            elif k == "tags":
+                self.assertTrue(bool(v), '"tags" must be `true` or a string of length>0')
                 if isinstance(v, str):
-                    self.assertFalse(v == "true",
-                                     'It is unlikely to specify the prefix '
-                                     '"true" use not the boolean `true`')
+                    self.assertFalse(
+                        v == "true",
+                        'It is unlikely to specify the prefix "true" use not the boolean `true`',
```

Comment:
Hello Yoda :D I had to read that 3 times ;)


```suggestion
                        '"true" is an unlikely prefix, you probably want the boolean `true` instead.',
```